nextflow-base-images/compliant_anibali_pytorch/Dockerfile

```diff
@@ -1,15 +0,0 @@
-# Use the specified base image
```

Comment:
please mention why this one is deleted somewhere in the PR description or a comment